api/src/main/resources/hibernate.cfg.xml

```diff
@@ -19,6 +19,7 @@
 	<session-factory>
         <property name="javax.persistence.validation.mode">none</property>
 		<property name="hibernate.search.model_mapping">org.openmrs.api.db.hibernate.search.LuceneAnalyzerFactory</property>
+		<property name="hibernate.hbm2ddl.auto">update</property>
```

Comment:
Does it help if you do this from? https://github.com/openmrs/openmrs-core/blob/master/api/src/main/resources/hibernate.default.properties

Reply:
I tried that by adding the property but that also didn't work. It is giving the same error as before:

`hibernate.hbm2ddl.auto=update`

`java.sql.SQLSyntaxErrorException: Table 'openmrs.REVINFO' doesn't exist`